mobile/cmp/popover/Popover.js

```diff
@@ -0,0 +1,196 @@
+/*
+ * This file belongs to Hoist, an application development toolkit
+ * developed by Extremely Heavy Industries (www.xh.io | info@xh.io)
+ *
+ * Copyright © 2021 Extremely Heavy Industries Inc.
+ */
+import {hoistCmp, HoistModel, useLocalModel} from '@xh/hoist/core';
+import {observable, action, makeObservable} from '@xh/hoist/mobx';
+import {div} from '@xh/hoist/cmp/layout';
+import {throwIf} from '@xh/hoist/utils/js';
+import {createObservableRef} from '@xh/hoist/utils/react';
+import {isNil, isFunction} from 'lodash';
+import {isValidElement} from 'react';
+import {usePopper} from 'react-popper';
+import ReactDom from 'react-dom';
+import classNames from 'classnames';
+import PT from 'prop-types';
+
+import './Popover.scss';
+
+/**
+ * Popovers display floating content next to a target element.
+ *
+ * The API is based on a stripped-down version of Blueprint's Popover component
+ * that is used on Desktop. Popover is built on top of the Popper.js library.
+ *
+ * @see https://popper.js.org/
+ */
+export const [Popover, popover] = hoistCmp.withFactory({
+    displayName: 'Popover',
+    model: false,
+    className: 'xh-popover',
+
+    render({
+        className,
+        target,
+        content,
+        isOpen,
```

Comment:
Seems like the big question is whether we like this style of using controlled mode, vs. exposing a model.